package.json

```diff
@@ -5,7 +5,8 @@
   "description": "NPM Packages maintained by Settlin",
   "main": ".eslintrc.js",
   "dependencies": {
-    "lodash": "^4.17.21"
+    "lodash": "^4.17.21",
```

Comment:
you can remove lodash from here - it is used in spacebars-loader package and is in its package.json already

Reply:
Resolved in new commit 
https://github.com/settlin/node-monorepo/pull/87/commits/41dd9265d855c4f96cf8ef3a5f63b8601039110d